### PR TITLE
ThredUp case study

### LINKED ISSUE #575: ThredUp Case Study
Build the ThredUp case study page.

**Design:** https://www.figma.com/file/q6VDs2NkDzmxXvivVpSI7o/ThredUP-Case-Study?node-id=0%3A1

**Google Doc:** https://docs.google.com/document/d/1mmG1Wx7EaqBiWIUcwzgui0_SG7L8Caq7vrGetsZArJI/edit#heading=h.6ef1ox49fmb1
(see **Rosalind's notes here**: https://github.com/gravitational/web/issues/2381)

**Assets/images/diagrams:** https://drive.google.com/drive/folders/1TOXESU375ZHaOuIah6gz2i7EggaptBFR
(please, **use the same images** as on the design)

**Icons:** https://drive.google.com/drive/folders/19RaYSzFKpSFdevS7GcFeXEP86gKdzfTW

same as: https://github.com/gravitational/web/issues/2381

### PR DESCRIPTION
**Pull request by C-STYR**
- Preview [here](https://teleport-docs-next-git-cole-thred-gravitational.vercel.app/case-study/thredup)
- fixes #575 
- fixes gravitational/next#2381
- finalizes layout in SimpleBanner

### CODE REVIEW

File: components/SimpleBanner/SimpleBanner.tsx [hunk 1]
```diff
@@ -77,22 +77,21 @@ export interface SimpleBannerProps {
...
+      alignItems={["center", "flex-start"]}
+      flexDirection="column"
+      textAlign="left"
+    >
+      <Flex
+        alignItems="flex-start"
+        flexDirection={["column", "row"]}
+        justifyContent="space-between"
+        mt={[3, 5]}
+        mb={[3, 5]}
```
Comment (sandylcruz) [inline components/SimpleBanner/SimpleBanner.tsx:90]: `my` to simplify

File: pages/case-study/thredup/index.mdx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  Link,
+  Feedback,
+  Quote,
+  SimpleBanner,
+  Section
+} from "components";
+
+<Section bg="flatWhite">
+  <SimpleBanner>
+    <SimpleBanner.Item isBadge src="building" title="Tied Access to Identity">
```
Comment (sandylcruz) [inline pages/case-study/thredup/index.mdx:33]: Are all the icons supposed to be of 'building'?

File: components/SimpleBanner/SimpleBanner.tsx (hunk 1 above)
Comment (C-STYR) [reply components/SimpleBanner/SimpleBanner.tsx:90]: nice, lol, was playing with those and failed to notice they ended up the same.

File: pages/case-study/thredup/index.mdx (hunk 2 above)
Comment (C-STYR) [reply pages/case-study/thredup/index.mdx:33]: certainly not!

File: components/SimpleBanner/SimpleBanner.tsx (hunk 1 above)
Comment (C-STYR) [inline components/SimpleBanner/SimpleBanner.tsx:90]:
```suggestion
        my={[3, 5]}
```

File: pages/case-study/thredup/index.mdx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+3. Kube-apiserver checks with an aws-iam-authenticator server running within the cluster.
+4. If the requested role is permissible, a valid kubeconfig file containing the role and permissions is generated for the user.
+
+<Figure
+caption="Figure 1: Access Workflow"
+align="center"
+>
+![Access Workflow](./assets/ThredUP_Image1.png)
+</Figure>
+
```
Comment (sandylcruz) [inline pages/case-study/thredup/index.mdx:91]: Is there supposed to be a video preview like in the mock up?

File: pages/case-study/thredup/index.mdx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+- Made onboarding and offboarding much simpler
+
+- Freed up vital company resources
+
+- Created a streamlined experience for developers
+
+thredUP continues to see added value from each new Teleport release, from access 
+workflow allowing administrators to grant real-time privileged access through Slack 
+or the Kubernetes enhancements that allow for `kubectl` events to be logged. thredUP 
+is now rolling out a Database Access solution following the success with Kubernetes and SSH Access.
```
Comment (sandylcruz) [inline pages/case-study/thredup/index.mdx:179]: Looks like there's supposed to be more space at the bottom

File: pages/case-study/thredup/index.mdx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+A user would access a Kubernetes cluster by following these steps:
+
+1. Client generates a token with the user's AWS credentials. Requested cluster name and role are included.
+2. Client presents the token to the kube-apiserver.
+3. Kube-apiserver checks with an aws-iam-authenticator server running within the cluster.
+4. If the requested role is permissible, a valid kubeconfig file containing the role and permissions is generated for the user.
+
+<Figure
+caption="Figure 1: Access Workflow"
+align="center"
```
Comment (sandylcruz) [inline pages/case-study/thredup/index.mdx:87]: Alignment should be to the left

Comment (C-STYR) [reply pages/case-study/thredup/index.mdx:87]: thank you, meant `imagePositioning` not `align`

File: pages/case-study/thredup/index.mdx (hunk 4 above)
Comment (C-STYR) [reply pages/case-study/thredup/index.mdx]: yup

**Review by sandylcruz — APPROVED** [review]

**Pull request merged**